plugins/performance-lab/includes/site-health/object-cache/helper.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Helper functions used for Object Cache Support Info.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```

Comment:
```suggestion
Per https://github.com/WordPress/performance/pull/1815:

```suggestion
// @codeCoverageIgnoreStart
if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly.
}
// @codeCoverageIgnoreEnd
```
```